addons/web/static/src/scss/attachment_preview.scss

```diff
@@ -0,0 +1,181 @@
+// Attachment Icons (common for chat thread and chat composer)
+// ------------------------------------------------------------------
+$o-mail-attachment-image-size: 38px;
+$o-mail-attachment-margin: 5px;
```

Comment:
those variables should be renamed to stop referencing mail (e.g. $o-mail-attachment-image-size -> $o-attachment-image-size)

Reply:
Done. renamed